docs-core/src/main/resources/db/update/dbupdate-028-0.sql

```diff
@@ -1,14 +1,4 @@
-create memory table T_REVIEW (
-    REV_USERNAME_C varchar(36),
-    REV_RATING_C integer,
-    REV_REVIEWID_C varchar(36) not null,
-    REV_USERID_C varchar(36),
-    REV_DOCID_C varchar(36),
-    REV_REVIEWERID_C varchar(36),
-    REV_CREATEDATE_D date not null,
-    REV_DELETEDATE_D date,
-    primary key (REV_REVIEWID_C)
-);
+create memory table T_REVIEW ( REV_USERNAME_C varchar(36), REV_RATING_C integer, REV_REVIEWID_C varchar(36) not null, REV_USERID_C varchar(36), REV_DOCID_C varchar(36), REV_REVIEWERID_C varchar(36), REV_CREATEDATE_D date not null, REV_DELETEDATE_D date, primary key (REV_REVIEWID_C) );
 
-insert into T_REVIEW(REV_USERNAME_C, REV_RATING_C, REV_REVIEWID_C, REV_USERID_C, REV_DOCID_C, REV_REVIEWERID_C, REV_CREATEDATE_D, REV_DELETEDATE_D)
-values ("John Doe", 5, "review123", "jdoe", "doc123", "reviewerperson", 09/28/2022, 09/28/2023);
\ No newline at end of file
+insert into T_REVIEW(REV_USERNAME_C, REV_RATING_C, REV_REVIEWID_C, REV_USERID_C, REV_DOCID_C, REV_REVIEWERID_C, REV_CREATEDATE_D, REV_DELETEDATE_D) values('John Doe', 5, 'review123', 'jdoe', 'doc123', 'reviewerperson', NOW(), NOW());
+insert into T_REVIEW(REV_USERNAME_C, REV_RATI`NG_C, REV_REVIEWID_C, REV_USERID_C, REV_DOCID_C, REV_REVIEWERID_C, REV_CREATEDATE_D, REV_DELETEDATE_D) values('Florentino Perez', 10, 'review1234', 'fperez', 'doc1234', 'reviewerperson', NOW(), NOW());
```

Comment:
Let's add a few more entries

Reply:
Some ideas for entries:
- same users
- same reviewer
- different date times